contracts/contract-config.js

```diff
@@ -10,6 +10,14 @@ module.exports = {
     userReplicaSetBootstrapAddress: null,
     registryAddress: null
   },
+  audius_dev: {
+    verifierAddress: '0xbbbb93A6B3A1D6fDd27909729b95CCB0cc9002C0',
+    blacklisterAddress: null,
+    bootstrapSPIds: [],
+    bootstrapSPDelegateWallets: [],
+    bootstrapSPOwnerWallets: [],
+    userReplicaSetBootstrapAddress: null
```

Comment:
Probably want `registryAddress` too that's written to and saved in the contract-predeployed container and written out to here. 
Reference the changes from contract-predeployed - or let me know and I can walk you through it